### PR TITLE
[KEYCLOAK-11457] Backport of ubi8-minimal changes

### PR DESCRIPTION
**Pull request by douglaspalmer**

### CODE REVIEW

**Comment by stianst** [comment]
Shouldn't we wait for https://github.com/keycloak/keycloak-containers/pull/217 to be resolved?

**Comment by abstractj** [comment]
@douglaspalmer I'm very confused. I could not spot the difference between this PR and https://github.com/keycloak/keycloak-containers/pull/217/. If there's some, why not address in the same PR?

**Comment by stianst** [comment]
@abstractj this PR is for REL branch which will be used for KC 7.0.1. Next release from master will be 8.

**Comment by abstractj** [comment]
Damn, I missed the branch. Sorry about that.

**Comment by stianst** [comment]
Not the first time - the branch is not obvious at all. I've merged PRs to maintenance branches in the past by mistake.

**Comment by abstractj** [comment]
@stianst should we merge this one too?

File: server/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ ARG KEYCLOAK_DIST=https://downloads.jboss.org/keycloak/$KEYCLOAK_VERSION/keycloa
 
 USER root
 
-RUN yum update -y && yum install -y epel-release git && yum install -y jq openssl which && yum clean all
```
Comment (gmanolache) [inline server/Dockerfile:20]: Looks like you removed jq, it was used for custom scripts parsing the external ip when binding public interfaces in aws.

Comment (abstractj) [reply server/Dockerfile:20]: @gmanolache please follow up on keycloak-user mailing list, we do not keep track of comments on closed pull requests.

**Pull request merged**